added colorsetting function

jo-mueller opened the pull request.

dstansby commented on src/napari_matplotlib/base.py:
```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
+        Apply napari-compatible colorscheme to the axes object.
+        """
+        if self.axes is None:
+            return 0
```
```suggestion
            return
```

dstansby commented on src/napari_matplotlib/base.py:
```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
+        """
+        if self.axes is None:
+            return 0
+                # changing color of axes background to napari main window color
```
```suggestion
        # changing color of axes background to napari main window color
```

dstansby commented on src/napari_matplotlib/base.py:
```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
+        if self.axes is None:
+            return 0
+                # changing color of axes background to napari main window color
+        self.canvas.figure.patch.set_facecolor("#262930")
```
Could you stick the napari window color in a variable so it's easier to re-use (something like `NAPARI_WINDOW_COLOR = "#262930"`

jo-mueller replied:
Fixed in e9f4345ad3e874d8489eb38efc25182f882fde13

dstansby commented on src/napari_matplotlib/base.py:
```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
+        self.axes.set_facecolor("#262930")
+
+        # changing colors of all axes
+        self.axes.spines["bottom"].set_color("white")
+        self.axes.spines["top"].set_color("white")
+        self.axes.spines["right"].set_color("white")
+        self.axes.spines["left"].set_color("white")
```
Might be easier to iterate through the spines (`for spine in self.axes.spines`) instead here?

dstansby reviewed (changes requested):
A couple of minor changes, and a suggestion for iterating through the spines. Otherwise looking 👍  (and sorry about how long it's taken me to get to this, if you want me to finish this off just let me know!)

dstansby commented:
Ooh also a rebase would be good to check that `pre-commit` is happy!

jo-mueller commented:
Hi @dstansby ,

thanks for the feedback, I incorporated your suggestions. The last commit was me not realizing that `set_tight_layout()` had been deprecated when I rebased my branch ^^"

dstansby reviewed (commented):
Looks 👍  - not sure this does fix https://github.com/matplotlib/napari-matplotlib/issues/36, as there hasn't been a `matplotlibrc` file added. That's fine for now, but I'll make sure that issue doesn't get closed when I merge this 😄
